lib/Runtime/Language/InterpreterLoop.inl

```diff
@@ -157,14 +157,25 @@ SWAP_BP_FOR_OPCODE:
                 return nullptr;
 
             case INTERPRETER_OPCODE::ExtendedOpcodePrefix:
+            case INTERPRETER_OPCODE::DblExtendedOpcodePrefix:
```

Comment:
I don't think this should increase code size for non-asm.js, because there shouldn't be any cases using the double extended macro (the only case would be default).